readme.txt

```diff
@@ -0,0 +1,27 @@
+=== Wikipedia Preview ===
+Contributors: Wikimedia Foundation
```

Comment:
This is meant to be the wordpress.org userid, let's update if needed once we create the official Wikimedia account

Reply:
Updating to '[wikimediafoundation](https://profiles.wordpress.org/wikimediafoundation/)' then, I understand it's the username that goes here: https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readmeheader-information